test/step-function.localstack.test.ts

```diff
@@ -1449,6 +1449,81 @@ localstackTestSuite("sfnStack", (testResource, _stack, _app) => {
     null,
     { a: "a" }
   );
+
+  test(
+    "destructure",
+    (parent) =>
+      new StepFunction(
+        parent,
+        "sfn",
+        async ({
+          a,
+          bb: { value: b },
+          c = "what",
+          arr: [d, , e, f = "sir", ...arrRest],
+          value,
+        }) => {
```

Comment:
Scanned through the tests, seems like we're missing initializers?

Reply:
`f = "sir"`

Reply:
Hurrr durrr